docs/libcurl/opts/CURLOPT_SSLVERSION.3

```diff
@@ -64,7 +64,7 @@ The MAX macros are not supported for SSL backends axTLS or wolfSSL.
 .IP CURL_SSLVERSION_MAX_DEFAULT
 The flag defines the maximum supported TLS version by libcurl, or the default
 value from the SSL library is used. libcurl will use a sensible default
-maximum, which was TLS 1.2 up to before 7.61.0 and is TLS 1.3 since then -
+maximum, which was TLS v1.2 up to before 7.61.0 and is TLS 1.3 since then -
```

Comment:
There's still a "TLS 1.3" here.

Reply:
Thanks, fixed in a to-be-squashed commit.